packages/waku/src/lib/plugins/vite-plugin-rsc-env.ts

```diff
@@ -11,6 +14,14 @@ export function rscEnvPlugin({
   return {
     name: 'rsc-env-plugin',
     config(viteConfig) {
+      if (isDev) {
+        dotenv.config({
+          path: ['.env.local', '.env'],
+          processEnv: (globalThis as any).__WAKU_PRIVATE_ENV__,
+          override: true,
```

Comment:
Oh, I see. This overrides the existing object.
Well, I guess there's a better solution, as now we don't have a worker.

Reply:
Hm, but, this is fine for now. Let's merge.